src/C-interface/dense/bml_add_dense_typed.c

```diff
@@ -39,14 +44,23 @@ void TYPED_FUNC(
     const double alpha,
     const double beta)
 {
-    REAL_T alpha_ = alpha;
-    REAL_T beta_ = beta;
     int myRank = bml_getMyRank();
 
     int nElems = B->domain->localRowExtent[myRank] * B->N;
     int startIndex = B->domain->localDispl[myRank];
     int inc = 1;
 
+#ifdef BML_USE_MAGMA
+    nElems = B->N * B->ld;
+    MAGMA_T alpha__ = MAGMACOMPLEX(MAKE) (alpha, 0.);
```

Comment:
Could you make the identifier names `alpha__` and `beta__` consistent in terms of the number of underscores? I don't have a preference regarding one or two underscores, I'll leave that choice up to you, but I would prefer it if the names were the same :smile:

Reply:
Replacing __ with _ in a few locations